docs/usage.md

```diff
@@ -47,11 +41,10 @@ Check the documentation for your handler of interest in [Handlers](handlers/over
 
         ::: my_package.my_module.MyClass
             handler: python
-            selection:
+            options:
```

Comment:
Mmm it's probably too late (and I'm commenting on implementation, not on the docs), but it sure would be nice to just not require the redundant `options` key; it's almost always going to be just this one key with its sub-keys. `handler` could just be reserved and popped before passing it.

Reply:
We still need to be able to differentiate the global-only options of the handler and the ones that can be overridden locally, but that would work for local options yes. Maybe this would feel a bit inconsistent though.

Reply:
Yeah I don't know... at first I was thinking that maybe it would be good to eliminate that distinction even for the main config. But that'd be too hacky, basically validation for the constructor's arguments would have to be dropped.

https://github.com/mkdocstrings/mkdocstrings/blob/5867162b2eebf4829d39b6fe74843b60ced05a30/docs/handlers/overview.md?plain=1#L200-L205

For reference, another config keys documentation https://mkdocstrings.github.io/crystal/configuration.html

Reply:
What if we allow both ways for local options? 

```
::: id
    options:
      foo: bar
```

```
::: id
    foo: bar
```

And if you want to change the handler, you must use the options key:

```
::: id
    handler: qux
    options:
      foo: bar
```

Reply:
Well, lets keep the options key mandatory for now. We can always change things later.